data/config/ndef.json

```diff
@@ -237,17 +213,58 @@
       { "field": "ethnicity", "name": "Ethnicity" },
       { "field": "strain", "name": "Strain" },
       { "field": "species", "name": "Genus species" },
-      { "field": "submitter_id", "name": "Submitter ID" }
+      { "field": "submitter_id", "name": "Submitter ID" },
+      { "field": "race", "name": "Race" },
+      { "field": "hiv_status", "name": "HIV Status" }
     ],
     "filterConfig": {
       "tabs": [{
         "title": "Subject",
-        "fields": ["ethnicity", "gender", "species"]
+        "fields": ["ethnicity", "gender", "species", "race"]
+      },
+      {
+        "title": "Diagnosis",
+        "fields": [
+          "arthxbase",
+          "bshbvstat",
+          "bshcvstat",
+          "cd4nadir",
+          "status",
+          "hiv_status"
+        ]
+      },
+      {
+        "title": "Drug Resistance",
+        "fields": [
+          "amikacin_res_phenotype",
+          "capreomycin_res_phenotype",
+          "isoniazid_res_phenotype",
+          "kanamycin_res_phenotype",
+          "ofloxacin_res_phenotype",
+          "pyrazinamide_res_phenotype",
+          "rifampicin_res_phenotype",
+          "rifampin_res_phenotype",
+          "streptomycin_res_phenotype"
+        ]
       },
       {
         "title": "Resource",
-        "fields": ["dataset", "research_focus"]
+        "fields": ["dataset"]
       }]
+    },
+    "tableConfig": {
+      "fields": [
+        "dataset",
+        "studyAccession",
+        "phenotype",
+        "gender",
+        "ethnicity",
+        "strain",
+        "species",
+        "submitter_id",
+        "race",
+        "hiv_status"
+      ]
```

Comment:
Can you update the ndef-qa.json file accordingly as well? I'd like to deploy this PR to qa-test3 to test it